geomstats/geometry/grassmannian.py

```diff
@@ -87,4 +83,4 @@ def exp(self, vector, point):
         """
         expm = gs.linalg.expm
```

Comment:
Do you want to make this `Matrices.expm` at some point? I have no strong opinion on this, it is also good not to re-implement numpy x)

Reply:
Yes! 
In the end I think this ought to point to `GL.exp` -- or even to `SO.exp` if we figure out what numpy does and does not do :)

(in methods like `linalg.svd` for instance, I had to remove the `hermitian=True` flag because unsupported in numpy 1.16.  It's not clear what numpy `expm` does as 1) we had to fix vectorization and 2) we don't know what algorithm it uses when given a symmetric matrix, as `expm` does not accept such flags as arguments, same for `logm`. This is related to @ythanwerdas change of the algorithm for SPD and it would be nice to have some benchmarks! in the meantime it will be good to have all the `expm` calls point to one common place so we can update these more easily)

Reply:
BTW I'll come with updates for the `grassmannian.py` PR but after GL and SO

Reply:
Numpy does not have expm, you're referring to scipy, right? Yes to have everything at a commong place for sure.

Reply:
Cool for GL, SO and the Grassmanians, but let's all agree on the inheritance structure first? cf #382

Reply:
Hmm...  it's still not so clear to me. I'll write more on that in 382 soon, see also the GL PR to come.